### PR TITLE
util/sys/linux: migrate test-infra to testify

### LINKED ISSUE #26422: migrate test-infra to testify for `util/sys/linux` pkg

### PR DESCRIPTION
**Pull request by unconsolable**
Signed-off-by: unconsolable <chenzhipeng2012@gmail.com>

### What problem does this PR solve?

Issue Number: close #26422 

### Release note 

- No release note

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
LGTM

File: util/sys/linux/sys_test.go [hunk 1]
```diff
@@ -10,25 +10,18 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package linux_test
```
Comment (xhebox) [inline util/sys/linux/sys_test.go:13]: I think it is better to keep the test binary a separated package whenever possible/feasible.

**Review by xhebox — APPROVED** [review]

**Review by tiancaiamao — APPROVED** [review]

**Pull request merged**